conformity/fields/email.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import, unicode_literals
+
+import re
+
+import attr
+
+from conformity.error import Error
+from conformity.fields.basic import UnicodeString
+from conformity.utils import strip_none
+from conformity.fields.net import IPAddress
```

Comment:
This is in the wrong place alphabetically. Please re-order.

Reply:
edited.